templater/templater.go

```diff
@@ -185,6 +186,15 @@ func templateFuncs(c *context.Context, rs *context.ResourceSet) template.FuncMap
 		b, _ := json.Marshal(data)
 		return string(b)
 	}
+
+	m["toYaml"] = func(v interface{}) string {
+		data, err := yaml.Marshal(v)
+		if err != nil {
+			// Swallow errors inside of a template.
+			return ""
```

Comment:
Template functions are allowed to return errors (see some of the functions below for example) and should do so - we don't want a situation in which templating silently fails.

Reply:
Right away ;)